test/smoke-limbo/veccopy-ompt-target-cmake/performBuildRun.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+usage() { echo "Usage: $0 [-a <AOMP directory>]" \
+          1>&2; exit 1; }
+
+while getopts ":Ga:" o; do
+    case "${o}" in
+        a)
+            AOMP_DIR=${OPTARG}
+            ;;
+        *)
+            usage
+            ;;
+    esac
+done
+
+# Set 'PROJECT_NAME' to the parent directory's name
+PROJECT_NAME=$(basename $(pwd))
+
+# If 'AOMP_DIR' was not specified fallback to user's AOMP directory
+if [[ -z ${AOMP_DIR} ]]; then
+  AOMP_DIR="/home/$USER/rocm/aomp"
+fi
+
+if [ ! -d ${AOMP_DIR} ]; then
+  echo "WARNING: AOMP directory '${AOMP_DIR}' does not exist!"
+fi
+
+echo " >>> Configure ..."
+cmake -B build -S . -DAOMP_DIR=${AOMP_DIR}
```

Comment:
Now it will, but when the default `native` is used, we will "offload" to x86_64.
Added `-t` (target offload arch) option to the bash script, which will in turn be provided via `make`.